docs/resources/compute_instance_v2.md

```diff
@@ -286,6 +286,9 @@ The following arguments are supported:
   to associate with the server. Changing this results in adding/removing
   security groups from the existing server.
 
+~> **Warning:** Names should be used and not IDs. Security group names should be **unique**
```

Comment:
Should be a `**Warning**` without a colon. Automatically generated `Note` has no colon.

Also, there should be a comma before `otherwise`